src/services/cli-http-client.js

```diff
@@ -140,6 +137,20 @@ class CliRequestClient {
     this.logger.debug(`User-Agent: ${options.headers['User-Agent']}`);
     this.logger.debug('-- END Twilio API Request --');
   }
+
+  /* eslint-disable camelcase */
+  // In the rare event parameters are missing, display a readable message
+  formatErrorMessage({ code, message, more_info, details }) {
```

Comment:
nit: remove space after `details`

Reply:
prettier wants the space after `details` otherwise, it blocks me from commiting